inference/environments/minimal-ubuntu20.04-py38-cpu-inference/context/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/azureml/inference-base-2004:{{latest-image-tag}}
+
+RUN conda create -n minimal \
+    python=3.8.13 pip=22.1.2
+
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /opt/miniconda/envs/minimal
```

Comment:
We've normally been putting these conda environments under `/azureml-envs`. Any concerns with continuing with this approach?

Reply:
I think to create the env under /azureml-envs requires to be root.

Reply:
It helps for us to be consistent where possible with our built-in environments. I'd recommend adding a `USER root:root` command to the Dockerfile and then generally using [this Dockerfile](https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/context/Dockerfile) as an example.

Reply:
Updated the approach.